docs/src/administration/backup-and-restore.md

```diff
@@ -87,6 +87,16 @@ Restoring a snapshot does not revert any code changes committed to git. The next
 
 A backup does cause a momentary pause in service. We recommend running during non-peak hours for your site.
 
+### Live backups
+
+There is an option available to create backups in a "live" state. This flavor leaves the environment running and open to connections, reducing downtime while the backup is taken. While use cases certainly exist, it is generally recommended to schedule [automated backups](#automated-backups) during non-peak hours in most cases, since a live backup may produce inconsistent states that are less reliable during restorations. 
```

Comment:
I'd reverse the last sentence here.  Lead with "there may be inconsistencies in the backup" therefore "we recommend using scheduled backups instead and just sucking it up on the downtime."